src/components/pacbio/PacbioRunWellDefaultEdit.vue

```diff
@@ -193,32 +188,12 @@ export default {
   name: 'PacbioRunWellDefaultEdit',
   data() {
     return {
-      movieTimeOptions: [
-        { text: 'Movie Time', value: '', disabled: true },
-        '10.0',
-        '15.0',
-        '20.0',
-        '24.0',
-        '30.0',
-      ],
+      movieTimeOptions: ['10.0', '15.0', '20.0', '24.0', '30.0'],
       generateHifiOptions: {
-        '': [{ text: 'Please select a System Name', value: '', disabled: true }],
-        'Sequel I': [
-          { text: 'Please select a value', value: '', disabled: true },
-          'In SMRT Link',
-          'Do Not Generate',
-        ],
-        'Sequel II': [
-          { text: 'Please select a value', value: '', disabled: true },
-          'In SMRT Link',
-          'Do Not Generate',
-        ],
-        'Sequel IIe': [
-          { text: 'Please select a value', value: '', disabled: true },
-          'In SMRT Link',
-          'Do Not Generate',
-          'On Instrument',
-        ],
+        '': [],
+        'Sequel I': ['In SMRT Link', 'Do Not Generate'],
```

Comment:
Nice clean up. I will check but have a feeling we can clean up further by remove Sequel I.

Reply:
Do you mean combining 'Sequel I' and 'Sequel II' as they both needs same options?